sourcecred/sourcecred
# Flow typed docs script

# Description
Dependant on #2435 
Flow typing the local generated docs script. Using the flow comment type structure because we need it to work as plain javascript.


# Test Plan
- Run `yarn docs` to generate the docs folder
- Ensure it logs `success` for all generated docs
